File: Python-Assignment/Chapter-Three/question20.py
binary_number = int(input("Enter a binary_number: "))

decimal_number = 0
count = 1

while binary_number > 0:
	digit = binary_number % 10

	decimal_number += digit * count

	count = count * 2

	binary_number = binary_number // 10

print(decimal_number)

File: Python-Assignment/chapterThree/Question20.java
import java.util.Scanner;

public class Question20{
   public static void main(String []args){

	Scanner input = new Scanner(System.in);

	System.out.print("Enter any binary number: ");
	int binaryNumber = input.nextInt();

	int decimal = 0;
	int count = 1;

	while(binaryNumber > 0){
		int digit = binaryNumber % 10;

		decimal = decimal + digit * count;

		count = count * 2;

		binaryNumber = binaryNumber / 10;
	}

	System.out.print(decimal);
  }
}